README.adoc

```diff
@@ -27,11 +29,30 @@ Updatecli reads a yaml or a go template configuration file, then works into thre
 
 == Why
 
-They are many projects out there, which can continuously update your files but they all have an opinionated way of doing it and they often want you to adopt a new platform. Building and distributing software is a difficult thing and good practices constantly evolve. Updatecli was built to work independently wherever you need, and combining whatever workflow you are using as you can see in the following section.
+They are many projects out there which can continuously update your files, but they all have an opinionated way of doing it and they often want you to adopt a new platform. Building and distributing softwares is a difficult task and good practices constantly evolve. Updatecli was built to work independently wherever you need, and combining whatever workflow you are using, as you can see in the following section.
```

Comment:
```suggestion
There are already many projects out there which can continuously update your files, but they all have an opinionated way of doing it and they often want you to adopt a new platform. 
Building and distributing software is a difficult task and good practices constantly evolve. 
Updatecli was built to work independently of the underlying dependencies to update, wherever you need it and combining whatever workflow you are using, as you can see in the following section.
```

Reply:
(One line per sentence is clearly easier to read and review in markdown/asciidoctor at least)